apps/backend/src/libs/plugins/authorization-plugin/authorization-plugin.ts

```diff
@@ -19,15 +19,17 @@ const authorizationPlugin = fp<PluginOptions>((app, options, done) => {
 	const { token, userService, whiteRoutes = [] } = options;
 
 	app.addHook(ServerHooks.PRE_HANDLER, async (request) => {
-		const isWhiteRoute = whiteRoutes.some((whiteRoute) =>
-			request.url.endsWith(whiteRoute),
-		);
+		const { headers, url } = request;
+
+		const isWhiteRoute = whiteRoutes.some((whiteRoute) => {
+			return url.startsWith("/api") && url.endsWith(whiteRoute);
```

Comment:
Use regex here. The url should match /api/v{number/endpoint, take endpoint from there and check if it is in white routes. Create a helper for that